src/ui/flightdisplay/QGCFlightDisplay.h

```diff
@@ -69,6 +70,7 @@ class QGCFlightDisplay : public QGCQmlWidgetHolder
     float   latitude            () { return _latitude; }
```

Comment:
I had only glanced at it and I had the impression it would allow the QLM side to write to these variables, which I don't usually want to allow. To be honest I've never paid much attention beyond the initial look up to see how to use these bindings. Only the other day I noticed what you did with the Main Tool bar. I thought about it last night but I didn't want to change that then. Something else for me to revisit.